src/__tests__/utils-test.ts

```diff
@@ -119,6 +120,7 @@ test('update navigation bar, without options', cb => {
 test('update navigation bar, with options', cb => {
     const options = {
         title: ANY_STRING,
+        expandedTitle: ANY_STRING,
```

Comment:
I'd use different strings, imagine some weird bug in the implementation where the expandedTitle is sent as title, this test wouldn't catch it

Reply:
Sure.